src/server/pegasus_mutation_duplicator.cpp

```diff
@@ -238,28 +239,41 @@ void pegasus_mutation_duplicator::duplicate(mutation_tuple_set muts, callback cb
     auto batch_request = std::make_unique<dsn::apps::duplicate_request>();
     uint batch_count = 0;
     uint batch_bytes = 0;
+    const static std::set<int> ingnored_rpc_code = {dsn::apps::RPC_RRDB_RRDB_DUPLICATE,
+                                                    dsn::apps::RPC_RRDB_RRDB_BULK_LOAD};
+
     for (auto mut : muts) {
         // mut: 0=timestamp, 1=rpc_code, 2=raw_message
         batch_count++;
         dsn::task_code rpc_code = std::get<1>(mut);
         dsn::blob raw_message = std::get<2>(mut);
         auto dreq = std::make_unique<dsn::apps::duplicate_request>();
 
-        if (rpc_code == dsn::apps::RPC_RRDB_RRDB_DUPLICATE) {
-            // ignore if it is a DUPLICATE
-            // Because DUPLICATE comes from other clusters should not be forwarded to any other
-            // destinations. A DUPLICATE is meant to be targeting only one cluster.
+        // ignore if it is a DUPLICATE or BULKLOAD
+        // Because DUPLICATE comes from other clusters should not be forwarded to any other
+        // destinations. A DUPLICATE is meant to be targeting only one cluster.
```

Comment:
```suggestion
        // The rpc codes should be ignored:
        // - RPC_RRDB_RRDB_DUPLICATE: Now not supports duplicating the deuplicate mutations to the remote cluster.
        // - RPC_RRDB_RRDB_BULK_LOAD: Now not supports the control flow RPC.
```
And move these before line 242.